rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -606,7 +607,23 @@ def upload_mutipart_object(
     # log.info('parts_info so far: %s'% parts_info)
     if len(parts_list) == part_number:
         log.info("all parts upload completed")
-        mpu.complete(MultipartUpload=parts_info)
+        if complete_abort_race:
+            # mpu.complete(MultipartUpload=parts_info)
```

Comment:
please remove commented line.

Reply:
removed the commented line
thanks for the review @ckulal